### PR TITLE
add Gemfile

### PR DESCRIPTION
**Pull request by ndushay**
first step in making this a deployable repo

- would you prefer one PR for all the setup, or a bunch of little ones?

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'byebug'
+gem 'pry-byebug'
+
+group 'deployment' do
+  gem 'dlss-capistrano', require: false
```
Comment (jcoyne) [inline Gemfile:9]: This is not a requirement of this repo. Each individual repo may have a different version of this.

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'byebug'
+gem 'pry-byebug'
+
+group 'deployment' do
```
Comment (jcoyne) [inline Gemfile:8]: why do we need a group?

File: Gemfile (hunk 1 above)
Comment (ndushay) [reply Gemfile:9]: I figured we would deploy THIS repo, too.

File: Gemfile (hunk 2 above)
Comment (ndushay) [reply Gemfile:8]: meh, probably don't.  Just doing the normal thing.

File: Gemfile (hunk 1 above)
Comment (ndushay) [reply Gemfile:9]: easiest way to keep it up to date is to treat it like all the other repos, no?

**Pull request merged**